.gitignore

```
@@ -0,0 +1,6 @@
+# dependencies
+/node_modules
+
```

Стоит добавить в игнор
```suggestion
.DS_Store
```